extensions/parquet/base/src/main/java/io/deephaven/parquet/base/PlainIntChunkedWriter.java

```diff
@@ -9,30 +9,30 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
-import java.nio.*;
+import java.nio.IntBuffer;
 
 
 /**
- * Plain encoding except for booleans
+ * Plain encoding except for ints
+ * TODO see if we can replicate from this, dhe seems to
```

Comment:
The first line of the comment is nonsense.
The second line is maybe unnecessary. We replicate this with `io.deephaven.replicators.ReplicateParquetChunkedWriters`, which is run from `:replication-static replicateParquetChunkedWriters`, apparently followed by manual fixups.

Reply:
This might be my attempt at keeping track what is happening in code that I don't understand, I'll remove these notes.